### ISSUE TITLE
Shellspec installation does not work on Windows platform

### ISSUE THREAD

**Issue by damienflament** [neimad/travis-python#4]
The build on Travis CI Windows platform fail due to an error with the TTY:

```console
$ curl -sSL https://git.io/shellspec | sh -s -- -y
Executable file        : /c/Users/travis/bin/shellspec
Installation directory : /c/Users/travis/lib/shellspec
Version (tag or commit): 0.20.2
[git] https://github.com/shellspec/shellspec.git

The command "curl -sSL https://git.io/shellspec | sh -s -- -y" failed and exited with 1 during .

Your build has been stopped.
sh: line 225: /dev/tty: No such device or address
Canceled
```

**Comment by damienflament** [comment]
See shellspec/shellspec#43.

**Comment by damienflament** [comment]
Implemented a workaround in 82ab5df : Shellspec is installed without using the installer.

Still need a fix from Shellspec.
